Common/Product/SharedProject/ProjectNode.IOleCommandTarget.cs

```diff
@@ -515,6 +517,11 @@ protected virtual int QueryStatusSelection(Guid cmdGroup, uint cCmds, OLECMD[] p
                 return VSConstants.S_OK;
             }
 
+            // This is not the right constant to return here, since we're
```

Comment:
This seems to match the documented interface: https://docs.microsoft.com/en-us/visualstudio/extensibility/internals/command-implementation.

I think the `OLECMDERR_` constants *are* hresults.